app/models/claims/state_machine.rb

```diff
@@ -145,7 +145,7 @@ def self.included(klass)
       end
 
       klass.has_many :archived_claim_state_transitions, lambda {
-        where(to: CASEWORKER_DASHBOARD_ARCHIVED_STATES)
+        where(to: CASEWORKER_DASHBOARD_ARCHIVED_STATES).order(created_at: :asc)
```

Comment:
I don't think you will need this but there may be a `default_scope` that needs unscoping `unscope(:order).order(created_at: :asc)`. Could not see any obvious one for claim_state_transitions though

Reply:
There is a scope that sets the order on the `claim_state_transitions` association in `Claim::BaseClaim` so I presume there isn't a default scope on the `ClaimStateTransition` model. I cannot see one either.